bins/revme/src/cmd/bench.rs

```diff
@@ -27,6 +30,7 @@ impl BenchName {
             BenchName::Burntpix => "burntpix",
             BenchName::Snailtracer => "snailtracer",
             BenchName::Transfer => "transfer",
+            BenchName::EvmBuild => "evm_build",
```

Comment:
```suggestion
            BenchName::EvmBuild => "evm-build",
```

For outside names using dash is better.

Reply:
great point @rakita, updated to: https://github.com/bluealloy/revm/blob/e537c96d8c224603eeccb2bfb2a62407bf5de7a2/bins/revme/src/cmd/bench.rs#L33

agreed that `cargo bench --package revme -- evm-build` does look cleaner